src/Mesh/Topography.jl

```diff
@@ -0,0 +1,163 @@
+module Topography
+#
```

Comment:
Can we change these comments to a module doc string?

Reply:
might leave it for now with fixed index if this is ok? this might be a module that lends itself to literate.jl documentation in the future